src/allmydata/storage/http_client.py

```diff
@@ -0,0 +1,77 @@
+"""
+HTTP client that talks to the HTTP storage server.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from future.utils import PY2
+
+if PY2:
+    # fmt: off
+    from future.builtins import filter, map, zip, ascii, chr, hex, input, next, oct, open, pow, round, super, bytes, dict, list, object, range, str, max, min  # noqa: F401
+    # fmt: on
+else:
+    from typing import Union
+    from treq.testing import StubTreq
```

Comment:
not clear why this is py3-only? "because only type-hints"..?

Reply:
`typing` module doesn't exist in Python 2, and we only do type checking in Python 3.